client/src/components/desk/DeskUtil.tsx

```diff
@@ -13,3 +15,10 @@ export function updateRequestStatus(updateRequest: any, request_id: number, newS
         }
     });
 }
+
+export function requestSearch(r: Request, searchQuery: string): boolean {
+    return r.user.name.toLowerCase().indexOf(searchQuery) >= 0
```

Comment:
there's a string `.includes` method that you may want to consider here

Reply:
😯 neato yes thx